fred2/mainfrm.cpp

```diff
@@ -77,6 +77,7 @@ static UINT indicators[] =
 
 CMainFrame *Fred_main_wnd;
 color_combo_box m_new_ship_type_combo_box;
+size_t num_ships_in_combo_box = 0;
```

Comment:
Should probably try to find a more specific name.... ship_type_combo_box_size ?

Reply:
Done and done